FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Constructor.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using FrontDesk.Core.Aggregates;
+using PluralsightDdd.SharedKernel;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Constructor
+  {
+    [Fact]
+    public void ConstructorCreatedAndIdAutoGenerated()
```

Comment:
The name says "AndIdAutoGenerated" but I don't see any assertions on appointment id.